examples/hotrod/cmd/root.go

```diff
@@ -73,6 +75,9 @@ func init() {
 	RootCmd.PersistentFlags().IntVarP(&frontendPort, "frontend-service-port", "f", 8080, "Port for frontend service")
 	RootCmd.PersistentFlags().IntVarP(&routePort, "route-service-port", "r", 8083, "Port for routing service")
 
+	// Flag for serving frontend at custom basepath url
+	RootCmd.PersistentFlags().StringVarP(&basepath, "basepath", "b", "/", "Basepath for frontend service")
```

Comment:
nit: suggest default of empty string `""`as we shouldn't expect the user to know to prefix with `/` but code should support both. See the suggestions below.